Mod fallingtree provides a broken implementation of ModMenuApi

### Describe the bug

When starting minecraft with the mod loaded this error pops up and the mod doesn't work!

### Minecraft version

1.18.1

### Fabric API version

0.46.0

### Mod version

3.1.0

### Relevant log output

```shell
java.lang.NoClassDefFoundError: me/shedaniel/autoconfig/gui/registry/GuiRegistry
	at fr.raksrinana.fallingtree.fabric.ModMenuImpl.<clinit>(ModMenuImpl.java:17)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:467)
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50)
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117)
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
	at com.terraformersmc.modmenu.ModMenu.lambda$onInitializeClient$0(ModMenu.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at com.terraformersmc.modmenu.ModMenu.onInitializeClient(ModMenu.java:67)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at net.minecraft.class_310.<init>(class_310.java:452)
	at net.minecraft.client.main.Main.main(Main.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.ClassNotFoundException: me.shedaniel.autoconfig.gui.registry.GuiRegistry
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 21 more
```

--- a/fabric/src/main/java/fr/raksrinana/fallingtree/fabric/ModMenuImpl.java
+++ b/fabric/src/main/java/fr/raksrinana/fallingtree/fabric/ModMenuImpl.java
@@ -5,17 +5,11 @@ import com.terraformersmc.modmenu.api.ModMenuApi;
 import fr.mrcraftcod.fallingtree.common.FallingTreeCommon;
 import fr.raksrinana.fallingtree.fabric.cloth.ClothConfigHook;
 import lombok.extern.slf4j.Slf4j;
-import me.shedaniel.autoconfig.gui.DefaultGuiProviders;
-import me.shedaniel.autoconfig.gui.DefaultGuiTransformers;
-import me.shedaniel.autoconfig.gui.registry.GuiRegistry;
 import net.fabricmc.loader.api.FabricLoader;
 import java.lang.reflect.InvocationTargetException;
 
 @Slf4j
 public class ModMenuImpl implements ModMenuApi{
-	@SuppressWarnings("unused")
-	private static final GuiRegistry defaultGuiRegistry = DefaultGuiTransformers.apply(DefaultGuiProviders.apply(new GuiRegistry()));
-	
 	@Override
 	public ConfigScreenFactory<?> getModConfigScreenFactory(){
 		if(FabricLoader.getInstance().isModLoaded("cloth-config")){

